python/cugraph/cugraph/internals/CMakeLists.txt

```diff
@@ -0,0 +1,35 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+set(cython_sources internals.pyx)
+set(linked_libraries cugraph::cugraph)
+rapids_cython_create_modules(
+  CXX
+  SOURCE_FILES "${cython_sources}"
+  LINKED_LIBRARIES "${linked_libraries}" MODULE_PREFIX internals_
+)
+
+set(targets_using_numpy internals_internals)
+
+include_directories(${CMAKE_CURRENT_LIST_DIR})
```

Comment:
What is this for?

Reply:
This should be a target based property I expect

Reply:
Addressed